+++ .bash_logout	2026-05-28 18:57:25.733498600 -0600
@@ -1,2 +1,22 @@
 # ~/.bash_logout
 
+LOGINCTL="$(which loginctl 2>/dev/null)"
+
+if [ -n "$LOGINCTL" ]
+then
+	NUMSESS="$($LOGINCTL | grep -c "$(whoami)")"
+	if [ -z "$TMUX" ] && [ "$NUMSESS" -eq 1 ]
+	then
+		if [ -S "$SSH_AUTH_SOCK" ]
+		then
+			for AGENT in $(pgrep ssh-agent)
+			do
+				kill -TERM "$AGENT"
+			done
+
+			rm -f "$SSH_AUTH_SOCK"
+		fi
+	fi
+fi
+
+clear
