+++ .bash_logout	2026-06-03 22:20:40.357706104 -0600
@@ -1,2 +1,23 @@
 # ~/.bash_logout
 
+LOGINCTL="$(which loginctl 2>/dev/null)"
+
+if [ -n "$LOGINCTL" ] && [ "$XDG_SESSION_TYPE" = "tty" ]
+then
+	NUMSESSIONS="$($LOGINCTL | grep -c "$(whoami)")"
+
+	if [ "$NUMSESSIONS" -eq 1 ] && [ -z "$TMUX" ]
+	then
+		if [ -S "$SSH_AUTH_SOCK" ]
+		then
+			for AGENT in $(pgrep ssh-agent)
+			do
+				kill -TERM "$AGENT"
+			done
+
+			rm -f "$SSH_AUTH_SOCK"
+		fi
+	fi
+fi
+
+clear
